led_raspberrypi_demo/README.md

```diff
@@ -0,0 +1,54 @@
+# Light Mapper
```

Comment:
rename the directory to led-raspberrypi ... no need to have demo at the end. Let us use - instead of _